doc/releases/pacific.rst

```diff
@@ -29,6 +29,148 @@ giant pacific octopus (Enteroctopus dofleini).
    DO NOT run ``ceph-bluestore-tool``'s repair/quick-fix commands.
 
 
+v16.2.7 Pacific   
+===============
+
+This is the seventh backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* Critical bug in OMAP format upgrade is fixed. This could cause data corruption
+  (improperly formatted OMAP keys) after pre-Pacific cluster upgrade if
+  bluestore-quick-fix-on-mount parameter is set to true or ceph-bluestore-tool's
+  quick-fix/repair commands are invoked.
+  Relevant tracker: https://tracker.ceph.com/issues/53062
+
+* MGR: The pg_autoscaler will use the 'scale-up' profile as the default profile.
+  16.2.6 changed the default profile to 'scale-down' but we ran into issues
+  with the device_health_metrics pool consuming too many PGs, which is not ideal
+  for performance. So we will continue to use the 'scale-up' profile by default,
+  until we implement a limit on the number of PGs default pools should consume,
+  in combination with the 'scale-down' profile.
+
```

Comment:
```suggestion
* Cephadm & Ceph Dashboard: NFS management has been completely reworked to
  ensure that NFS exports are managed consistently across the different Ceph
  components. Prior to this, there were 3 incompatible implementations for
  configuring the NFS exports: Ceph-Ansible/OpenStack Manila, Ceph Dashboard and
  'mgr/nfs' module. With this release the 'mgr/nfs' way becomes the official
  interface, and the remaining components (Cephadm and Ceph Dashboard) adhere to
  it. While this might require manually migrating from the deprecated
  implementations, it will simplify the user experience for those heavily
  relying on NFS exports.

* Dashboard: "Cluster Expansion Wizard". After the 'cephadm bootstrap' step,
  users that log into the Ceph Dashboard will be presented with a welcome
  screen. If they choose to follow the installation wizard, they will be guided 
  through a set of steps to help them configure their Ceph cluster: expanding
  the cluster by adding more hosts, detecting and defining their storage
  devices, and finally deploying and configuring the different Ceph services.
  
```